styles.css

```diff
@@ -301,4 +297,92 @@
     color: #253858;
     padding-left: 10px;
   }
+
+  /* CONTACT SECTION */
+  .contactFormSection {
+    background-image: url("./media-library/Contact-me.png");
+    height: 950px;
+    background-size: cover;
+    background-repeat: no-repeat;
+    display: grid;
+    grid-template-columns: 1fr;
+    padding: 0 24px;
+  }
+
+  .formHeadingContainer {
```

Comment:
- [ ] Please adhere to the `Figma` design as closely as possible. Your design has some small issues which make your design looks unprofessional. Kindly fix the spacing issue between the child element of this container, you can have a look at the below image.

<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-09 21-33-19](https://github.com/geekelo/a-personal-portfolio-site/assets/66476329/37d4c37a-15ec-4210-a975-655142894706)

</td>
<td>

![Screenshot from 2023-05-09 21-33-57](https://github.com/geekelo/a-personal-portfolio-site/assets/66476329/3c6338b0-73cd-422f-a78c-75b08cadd438)

</td>
</tr>
</table>